.styleguide

```diff
@@ -10,6 +10,8 @@ cppSrcFileInclude {
 
 generatedFileExclude {
   gtest/
+  simulation/gz_msgs/src/include/simulation/gz_msgs/msgs.h
```

Comment:
Please lexographically sort these.

Reply:
Also, .styleguide uses regular expressions. This should be `simulation/gz_msgs/src/include/simulation/gz_msgs/msgs\.h$`.